spark/v3.0/spark/src/test/java/org/apache/iceberg/spark/TestSparkCatalogOperations.java

```diff
@@ -77,4 +78,22 @@ public void testAlterTable() throws NoSuchTableException {
         propsValue,
         table.properties().get(propsKey));
   }
+
+  @Test
+  public void testInvalidateTable() {
+    if (!catalogName.equals(SparkCatalogConfig.HIVE.catalogName())) {
```

Comment:
You can use Assume in tests instead for better tracking.

Reply:
Fixed. Just use `validationCatalog` as another catalog.